src/pages/Cart/Cart.js

```diff
@@ -1,8 +1,150 @@
-import React from 'react'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useState, useEffect } from 'react'
+import { useNavigate } from 'react-router-dom'
+import Count from './Count/Count'
+import Modal from './Modal/Modal'
+import './Cart.scss'
 
 const Cart = () => {
-  return <h1>hello</h1>
+  const navigate = useNavigate()
+
+  const [cartData, setCartData] = useState([])
+  const [checkItems, setCheckItems] = useState([])
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  useEffect(() => {
+    fetch('/data/product.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/product.json')
```
단순 GET 요청일 때는 method를 사용하지 않으셔도 됩니다!